.changes/v3.6.0/812-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fix bug in **datasource/vcd_nsxt_app_port_profile** where TENANT scoped lookup could fail
+  findining exact Application Port Profile when Org has multiple VDCs [GH-812]
```

Comment:
```suggestion
* Fix bug in **datasource/vcd_nsxt_app_port_profile** where a lookup of a TENANT scope could fail
  finding exact Application Port Profile in case Org has multiple VDCs [GH-812]
```

Reply:
Changed this one.